### PR TITLE
MLW-917: Add Sputum patient

### PR DESCRIPTION
**Pull request by jomadoye**

### CODE REVIEW

File: src/screening/eid/EidSummary.jsx [hunk 1]
```diff
@@ -1,13 +1,10 @@
...
-import PatientLabTests from "../../patient/PatientLabTests";
-import {CONCEPTS} from "../../constants";
+import { ObsHistory } from '@openmrs/react-components';
+import { CONCEPTS } from "../../constants";
 
 const EidSummary = props => {
   return (
     <span>
-      <PatientLabTests test_type={ "viral_load_tests" }/>
-      <PatientLabTests test_type={ "hiv_tests" }/>
```
Comment (mogoodrich) [inline src/screening/eid/EidSummary.jsx]: @jomadoye  is this related to this pull request, or for something else?

Comment (jomadoye) [reply src/screening/eid/EidSummary.jsx]: oh, I mistakenly deleted that, updated.

File: src/screening/sputum/SputumForm.jsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+          >
+            Sputum received
+          </Col>
+        </Row>
+        <Row>
+          <Col sm={12}>
+            <FormGroup controlId="formBled">
+              <Obs
+                concept={CONCEPTS.SputumReceived.uuid}
+                conceptAnswers={FORM_ANSWERS.trueFalse}
```
Comment (jomadoye) [inline src/screening/sputum/SputumForm.jsx]: @mogoodrich , HERE I had to use the Yes/No concept Answers and one you posted on the ticket. That one wasn't working. Is this fine?

Comment (mogoodrich) [reply src/screening/sputum/SputumForm.jsx]: @jomadoye yes, that is correct, to use the true/false ones... I will comment to Dave on the ticket

**Pull request merged**